src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/DerivedKeySecurityToken.cs

```diff
@@ -0,0 +1,291 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+using CoreWCF.IdentityModel;
+using CoreWCF.IdentityModel.Tokens;
+
+namespace CoreWCF.Security.Tokens
+{
+    sealed class DerivedKeySecurityToken : SecurityToken
+    {
+        //        public const string DefaultLabel = "WS-SecureConversationWS-SecureConversation";
```

Comment:
Remove commented line. Also class is missing accessor as well as many of it's members.